newspaper/article.py

```diff
@@ -565,3 +565,9 @@ def throw_if_not_parsed_verbose(self):
         """
         if not self.is_parsed:
             raise ArticleException('You must `parse()` an article first!')
+
+    def yake(self, lan="en", n=3, dedupLim=0.9, dedupFunc='seqm', windowsSize=1, top=20, features=None):
+        import yake
```

Comment:
I think in-function imports are normally not preferred in python but in this case it's probably better incase yake installation fails we won't break the whole library, just this function call